### ISSUE TITLE
Add Line Items, Shipping address, etc

### ISSUE THREAD

**Issue by Sarke** [thephpleague/omnipay-authorizenet#107]
Is there a way to use any of the additional request properties, such as order line items and shipping address?  Those defined here: https://developer.authorize.net/api/reference/#payment-transactions

If not, is this going to be added?  Without it, it is very limited functionality.

It would be a good idea to have a method to add generic properties into the request, so we are not limited to what is specifically defined in `AIMAbstractRequest`.

**Comment by Sarke** [comment]
It appears that only the `CreditCard` class is used, the `ItemBag` is not.

As suggested, making `AbstractRequest::setParameter()` public would allow us to use payment gateway features without it being implemented in OmniPay.

**Comment by judgej** [comment]
Yes you are right, the `lineItems`/`ItemBag` seems to have been totally overlooked.
